code/game/objects/structures/electricchair.dm

```diff
@@ -1,6 +1,6 @@
 /obj/structure/chair/e_chair
 	name = "electric chair"
-	desc = "Looks absolutely SHOCKING!"
```

Comment:
I like the old description more here. Most people know what an electric chair is, we can afford a little joke

Reply:
Fairs.